modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheAdapter.java

```diff
@@ -2588,6 +2630,9 @@ public boolean put(final K key, final V val, final CacheEntryPredicate filter)
         if (statsEnabled && stored)
             metrics0().addPutTimeNanos(System.nanoTime() - start);
 
+        if (performanceStatsEnabled)
```

Comment:
Typo: let's use `performanceStatsEnabled` name here and in all other cases.

Reply:
Fixed.